packages/gluestick/src/generator/templates/EntryWrapper.js

```diff
@@ -31,12 +33,12 @@ export default class EntryWrapper extends Component {
 }
 
 // This function is called only on client.
-EntryWrapper.start = (getRoutes, getStore, match = originalMatch, history = browserHistory) => {
+const start = (getRoutes, getStore, match = originalMatch, history = browserHistory) => {
   // Allow developers to include code that will be executed before the app is
   // set up in the browser.
   require("config/init.browser");
 
-  const httpClient = getHttpClient(config.httpClient);
```

Comment:
I know that you removed `config.httpClient`. I'm also aware that in `config`, `httpClient` is not defined, but some existing apps might use it.